zoom_monitor.py

```diff
@@ -179,9 +169,11 @@ def _open_participants_list(self):
             for outer in self._driver.find_elements(
                     By.CLASS_NAME, "footer-button-base__button"):
                 try:
-                    self._logger.debug(f"trying to find participants in {outer}")
+                    self._logger.debug(
+                        f"trying to find participants default in {outer}")
                     # Check if this footer button contains the participants
-                    outer.find_element(By.CLASS_NAME, "SvgParticipantsDefault")
+                    outer.find_element(
+                        By.XPATH, "//*[contains(@class, 'SvgParticipants')]")
```

Comment:
Our xpath identifier is repeated twice (once here, once on line 152). Maybe we should make it a named constant, and then if we need to change it to, like, `startswith` instead of `contains` or specify the suffixes or anything, we only have one place to fix up.